src/utils/hooks/useCreateFinalDecisionReview.ts

```diff
@@ -24,48 +18,41 @@ type ConstructReviewPatch = (structure: FullStructure) => ReviewPatch
 const useCreateFinalDecisionReview: UseCreateFinalDecisionReview = ({
   reviewStructure,
   reviewAssignment,
-  previousAssignment,
 }) => {
   const [createReview] = useCreateReviewMutation()
 
   const getFullReviewStructureAsync = useGetFullReviewStructureAsync({
     reviewStructure,
     reviewAssignment,
-    previousAssignment,
   })
 
-  const shouldRemakeFinalDecisionReviewResponse = (element: PageElement) => {
-    if (previousAssignment?.level === 1) return true
-    return element?.thisReviewLatestResponse?.decision === ReviewResponseDecision.Agree
-  }
-
   const constructReviewPatch: ConstructReviewPatch = (structure) => {
     const elements = Object.values(structure?.elementsById || {})
 
     // Exclude not assigned, not visible and missing responses
     const reviewableElements = elements.filter((element) => {
       const { isAssigned, isActiveReviewResponse } = element
-      return (
-        shouldRemakeFinalDecisionReviewResponse(element) && isAssigned && !isActiveReviewResponse
-      )
+      return isAssigned && !isActiveReviewResponse
     })
 
-    // For re-assignment this would be slightly different, we need to consider latest review response of this level
-    // not necessarily this thisReviewLatestResponse (would be just latestReviewResponse, from all reviews at this level)
+    // Generate each reviewResponse for FinalStage review as a copy of previous Lower Level review
```

Comment:
> Generate each reviewResponse for FinalStage review 

Nit picky, but I think it'd be good to call this a "MakeDecision" review, as that's what its called in the UI. "Final Stage" is very much just an arbitrary name we give to the last stage. And this process you're describing here doesn't happen for every "final stage", only ones that have the "make decision" switch enabled, right?

Reply:
Hmmm that's a fair point. I'll rename functions/comments now